### ISSUE TITLE
Preprocessing for Fine-tuning

### ISSUE THREAD

**Issue by evanrubel** [microsoft/BiomedParse#19]
Thank you all for this excellent work and for open-sourcing it as well!

I am interested in fine-tuning the Biomedparse model for a CT dataset. In the inference code, there are some preprocessing steps for CT scans (e.g., windowing, normalization, etc.). When preparing the dataset for finetuning, should the input images in the `train` and `test` directories already be preprocessed, or will the training and evaluation code automatically preprocess the data?

**Comment by theodore-zhao** [comment]
They should be preprocessed before putting in the train and test folders. If there are image types that are out of the current preprocessing categories, feel free to preprocess in the way that makes most sense for your application. The model should be able to adapt to that after finetuning.
